junit-jupiter-engine/src/test/java/org/junit/jupiter/api/AssertionsAssertArrayEqualsTests.java

```diff
@@ -1874,8 +1876,13 @@ void assertArrayEqualsDifferentNestedObjectArrays() {
 			expectAssertionFailedError();
 		}
 		catch (AssertionFailedError ex) {
+
 			assertMessageEquals(ex,
-				"array contents differ at index [3][0][1][0], expected: <5> but was: <" + differentElement + ">");
+				"array contents differ at index [3][0][1][0], expected: <5> but was: <"
+						// @sbrannen, @marcphillip: Do we want to display the
+						// default array str repr e.g. [I@3343c8b, or a
+						// list-like repr e.g. [1, 2, 3]?
+						+ Arrays.toString(differentElement) + ">");
```

Comment:
@sbrannen The reason for this change was because error-prone reported the following compilation error at this line:

```
C:\Users\Jonathan\dev\Java\IntelliJ Projects\junit5\junit-jupiter-engine\src\test\java\org\junit\jupiter\api\AssertionsAssertArrayEqualsTests.java:1881: error: [ArrayToString] Calling toString on an array does not provide useful information
				"array contents differ at index [3][0][1][0], expected: <5> but was: <" + differentElement + ">");
				                                                                          ^
    (see http://errorprone.info/bugpattern/ArrayToString)
  Did you mean '"array contents differ at index [3][0][1][0], expected: <5> but was: <" + Arrays.toString(differentElement) + ">");'?
```

I could move it to a new PR, but like with https://github.com/junit-team/junit5/pull/961/files#r134096057, Travis CI won't go green until the new PR is merged and this PR is rebased on top of master.